L1Trigger/TrackFindingTMTT/interface/TrackerModule.h

```diff
@@ -134,6 +128,8 @@ namespace tmtt {
     unsigned int moduleTypeID_;
 
     ModuleTypeCfg moduleTypeCfg_;
+
+    float invRoot12 = sqrt(1./12.);
```

Comment:
const

Reply:
constants declared like this in header file can't be declared "const", or it disables the default copy constructor.

Reply:
`static constexpr` ?

Reply:
`const` member data disables *any* copy constructor. This one can't be declared as `constexpr` because `std::sqrt()` is not `constexpr`, unless the value is calculated manually and the result put here (with a comment what it is). On the other hand `static const` would work directly.

Reply:
Interesting... GCC allows `constexpr`, clang does not.
`static const` then.

Reply:
done